tests/bots.py

```diff
@@ -49,7 +49,10 @@
 BOTS = os.getenv('BOTS', None)
 JOB_INDEX = os.getenv('JOB_INDEX', None)
 if GITHUB_ACTION is not None and BOTS is not None and JOB_INDEX is not None:
-    BOTS = json.loads(base64.b64decode(BOTS).decode('utf-8'))
+    try:
+        BOTS = json.loads(base64.b64decode(BOTS).decode('utf-8'))
+    except UnicodeDecodeError:
+        BOTS = json.loads(base64.b64decode(BOTS).decode('utf-8', 'ignore'))
```

Comment:
please revert. not needed in this context.

Reply:
Got it.😋